pep-0548.rst

```diff
@@ -0,0 +1,338 @@
+published in 2012 (PEP 394). They differ mainly in granting platform providers
+more scope to effectively manage the eventual removal of their Python 2.7
+stacks.
+
+
+Recommendations for Development of Standalone Scripts
+=====================================================
+
+* standalone scripts that are incompatible with Python 2.x should specifically
+  target ``python3``.
+* standalone scripts that are incompatible with Python 3.x, and do not need to
+  gain compatibility with platforms that only provide Python 3.x, should
+  specifically target ``python2`` (as this is available by default almost
+  everywhere that Python 2 is available, and is easily added as a symlink if
+  it is missing).
+* standalone scripts that want to support both older platforms where ``python``
+  refers to ``python2`` *and* newer platforms or virtual environments where it
+  refers to ``python3`` should be updated to run in the common subset of Python
+  2.7 & 3.6+ using either the `regular porting guide`_ in the Python reference
+  documentation or else the `conservative porting guide`_ developed as part of
+  Fedora's migration to using Python 3 as the system Python runtime.
+* standalone scripts designed to run under both Python 2 and 3 may reasonably
+  require the availability of the ``six`` compatibility library. However,
+  avoiding this runtime dependency will improve compatibility with older
+  platforms that don't provide that component by default.
+* These distinctions should be made in shebang lines for Python scripts, when
+  invoking Python from shell scripts, when invoking Python via the C
+  ``system()`` call, or when invoking Python in any other context.
+* When reinvoking the interpreter from a Python script, querying
+  ``sys.executable`` to avoid hardcoded assumptions regarding the
+  interpreter location remains the preferred approach.
+
+.. _regular porting guide: https://docs.python.org/3/howto/pyporting.html
+.. _conservative porting guide: https://portingguide.readthedocs.io/en/latest/process.html
+
+
+Recommendations for Development of Cross-Platform Applications
+==============================================================
+
+* cross-platform applications should use appropriate mechanisms (such as Python
+  virtual environments or system level application containers) to avoid
+  depending on the default version of Python installed on any particular
+  deployment platform.
+
+
+Recommendations for Platform Providers
+======================================
+
+* Unix-like software distributions (including systems like Mac OS X and
+  Cygwin) should install the ``python2`` command into the default path
+  whenever a version of the Python 2 interpreter is installed, and the same
+  for ``python3`` and the Python 3 interpreter.
+* When  invoked, ``python2`` should run some version of the Python 2
+  interpreter, and ``python3`` should run some version of the Python 3
+  interpreter.
```

Comment:
Python 2.6 is done, and there will never be a Python 2.8, so let's just say "Python 2.7" here.

Also, I think you suggested elsewhere to floor the Python 3 version on 3.5, which I support.